app/Agri.Models/Farm/Field.cs

```diff
@@ -19,5 +19,7 @@ public Field()
         public string prevYearManureApplicationFrequency { get; set; }
         public int? prevYearManureApplicationNitrogenCredit { get; set; }
         public decimal? SoilTestNitrateOverrideNitrogenCredit { get; set; }
+        public bool IsSeasonalFeedingArea { get; set; }
+        public string SeasonalFeedingArea { get; set; }
     }
 }
```

Comment:
I also forgot to mention I am slowly cleaning up all old code when I come across it, so for classes like this one, please refactor the Property names to start with an Upper case.